SetupConfigure.cmake

```diff
@@ -243,11 +251,7 @@ endif()
 set(QT_SUPPORT ON)
 
 if (MUSE_MODULE_AUDIO_JACK)
-    if (OS_IS_LIN OR MINGW)
```

Comment:
What's going on here?

Reply:
But something is wrong. I guess it should be
```cmake
if (NOT (OS_IS_LIN OR MINGW))
     set(MUSE_MODULE_AUDIO_JACK OFF)
endif()

Reply:
I decided that jack is cross-platform, why do we limit it to Linux only... if it doesn't work somewhere, then we can just not enable it (it's disabled by default)

Reply:
Agree that practically it doesn't matter. But that change doesn't belong in this commit with that commit title, so it is strange why it was allowed.

The information removed is useful for a future jack revival/rehaul of the audio/midi system.

Reply:
But @igorkorsukov look at the new code:
```cmake
if (MUSE_MODULE_AUDIO_JACK)
     set(MUSE_MODULE_AUDIO_JACK OFF)
```
it always sets the option to OFF, even if it was enabled manually. That can't be the intention, is it?

Reply:
yes, that's right, it's a mistake, I'll fix it